content/ko/docs/concepts/configuration/secret.md

```diff
@@ -0,0 +1,1268 @@
+    - ls
+    - "-l"
+    - "/etc/secret-volume"
+    volumeMounts:
+    - name: secret-volume
+      readOnly: true
+      mountPath: "/etc/secret-volume"
+```
+
+볼륨은 `.secret-file` 이라는 하나의 파일을 포함하고,
+`dotfile-test-container` 는 `/etc/secret-volume/.secret-file` 경로에
+이 파일을 가지게 된다.
+
+{{< note >}}
+`ls -l` 명령의 결과에서 숨겨진 점으로 시작하는 파일들은
+디렉터리 내용을 나열할 때 `ls -la` 를 사용해야 이 파일들을 볼 수 있다.
+{{< /note >}}
+
+### 사용 사례: 파드의 한 컨테이너에 표시되는 시크릿
+
+HTTP 요청을 처리하고, 복잡한 비즈니스 로직을 수행한 다음, HMAC이 있는 일부 메시지에
+서명해야 하는 프로그램을 고려한다. 애플리케이션 로직이
+복잡하기 때문에, 서버에서 눈에 띄지 않는 원격 파일 읽기 공격이
+있을 수 있으며, 이로 인해 프라이빗 키가 공격자에게 노출될 수 있다.
+
+이는 두 개의 컨테이너의 두 개 프로세스로 나눌 수 있다. 사용자 상호 작용과
+비즈니스 로직을 처리하지만, 프라이빗 키를 볼 수 없는 프론트엔드 컨테이너와
+프라이빗 키를 볼 수 있고, 프론트엔드의 간단한 서명 요청(예를 들어, localhost 네트워킹을 통해)에
+응답하는 서명자 컨테이너로 나눌 수 있다.
+
+이 분할된 접근 방식을 사용하면, 공격자는 이제 애플리케이션 서버를 속여서
+파일을 읽는 것보다 다소 어려운 임의적인 어떤 작업을 수행해야
+한다.
+
+<!-- TODO: explain how to do this while still using automation. -->
+
+## 모범 사례
+
+### 시크릿 API를 사용하는 클라이언트
+
+시크릿 API와 상호 작용하는 애플리케이션을 배포할 때, [RBAC](
+/docs/reference/access-authn-authz/rbac/)과 같은 [인가 정책](
+/docs/reference/access-authn-authz/authorization/)을
+사용하여 접근를 제한해야 한다.
+
+시크릿은 종종 다양한 중요도에 걸친 값을 보유하며, 이 중 많은 부분이
+쿠버네티스(예: 서비스 어카운트 토큰)와 외부 시스템으로 단계적으로
+확대될 수 있다. 개별 앱이 상호 작용할 것으로 예상되는 시크릿의 힘에 대해 추론할 수 있더라도
+동일한 네임스페이스 내의 다른 앱이 이러한 가정을
+무효화할 수 있다.
+
+이러한 이유로 네임스페이스 내 시크릿에 대한 `watch` 와 `list` 요청은
+매우 강력한 기능이며, 시크릿을 나열하면 클라이언트가 해당 네임스페이스에
+있는 모든 시크릿의 값을 검사할 수 있기 때문에 피해야 한다. 클러스터의
+모든 시크릿을 감시(`watch`)하고 나열(`list`)하는 기능은 가장 특권이 있는 시스템 레벨의
+컴포넌트에 대해서만 예약되어야 한다.
+
+시크릿 API에 접근해야 하는 애플리케이션은 필요한 시크릿에 대한 `get` 요청을
+수행해야 한다. 이를 통해 관리자는 앱에 필요한
+[개별 인스턴스에 대한 접근을 허용 목록에 추가](
+/docs/reference/access-authn-authz/rbac/#referring-to-resources)하면서 모든 시크릿에 대한 접근을
+제한할 수 있다.
+
+`get` 반복을 통한 성능 향상을 위해, 클라이언트는 시크릿을
+참조한 다음 리소스를 감시(`watch`)하고, 참조가 변경되면 시크릿을 다시 요청하는 리소스를
+설계할 수 있다. 덧붙여, 클라이언트에게 개별 리소스를 감시(`watch`)하도록 하는 ["대량 감시" API](
+https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/bulk_watch.md)도
+제안되었으며, 향후 쿠버네티스 릴리스에서 사용할 수
+있을 것이다.
+
+## 보안 속성
+
+### 보호
+
+시크릿은 시크릿을 사용하는 파드와 독립적으로 생성될 수
+있으므로, 파드 생성, 보기, 편집 워크플로 중에
+시크릿이 노출될 위험이 적다. 또한 시스템은 가능한 경우
+디스크에 기록하지 않는 등 시크릿에 대한 추가 예방 조치를
+취할 수 있다.
+
+해당 노드의 파드에 필요한 경우에만 시크릿이 노드로 전송된다.
+kubelet은 시크릿이 디스크 저장소에 기록되지 않도록 시크릿을
+`tmpfs` 에 저장한다. 일단 시크릿에 의존하는 파드가 삭제되면, kubelet은
+시크릿 데이터의 로컬 복제본도 삭제한다.
+
+동일한 노드의 여러 파드에 대한 시크릿이 있을 수 있다. 그러나
+파드가 요청하는 시크릿만 해당 컨테이너 내에서 잠재적으로 볼 수 있다.
+따라서, 하나의 파드는 다른 파드의 시크릿에 접근할 수 없다.
+
+파드에는 여러 개의 컨테이너가 있을 수 있다. 그러나, 파드의 각 컨테이너는
+컨테이너 내에서 볼 수 있도록 파드의 `volumeMounts` 에 있는 시크릿 볼륨을
+요청해야 한다. 이것은 유용한 [파드 레벨에서의 보안
+파티션](#사용-사례-파드의-한-컨테이너에-표시되는-시크릿)을 구성하는 데 사용할 수 있다.
+
+대부분의 쿠버네티스 배포판에서, 사용자와 API 서버 간,
+API 서버에서 kubelet으로의 통신은 SSL/TLS로 보호된다.
+이러한 채널을 통해 전송될 때 시크릿이 보호된다.
+
+{{< feature-state for_k8s_version="v1.13" state="beta" >}}
+
+시크릿 데이터에 대해 [저장 시 암호화(encryption at rest)](/docs/tasks/administer-cluster/encrypt-data/)를
+활성화할 수 있으므로, 시크릿이 {{< glossary_tooltip term_id="etcd" >}}에 확실히 저장되지 않는다.
```

Comment:
```suggestion
시크릿 데이터에 대해 [저장 시 암호화(encryption at rest)](/docs/tasks/administer-cluster/encrypt-data/)를
활성화할 수 있으며, 이를 통해 보안성에 대한 보장 없이는 시크릿이 {{< glossary_tooltip term_id="etcd" >}}에 저장되지 않도록 한다 .
```
encryption at rest 는 etcd 에 저장시 암호화를 적용하는 것이라고 이해하였습니다.
이를 고려하면 번역이 원문과 의미가 다소 다르게 느껴집니다. 참고하시기 바랍니다.

```
You can enable [encryption at rest](/docs/tasks/administer-cluster/encrypt-data/)
for secret data, so that the secrets are not stored in the clear into {{< glossary_tooltip term_id="etcd" >}}.
```

Reply:
반영하겠습니다.